processing/src/main/java/org/apache/druid/query/aggregation/last/StringLastAggregatorFactory.java

```diff
@@ -302,6 +303,7 @@ public String toString()
            "fieldName='" + fieldName + '\'' +
```

Comment:
It is a good idea to autofill these will the IDE. 1 minor change 
```
", maxStringBytes=" + maxStringBytes + '\'
```
Also, does the hashcode() need a change for this as well?

Reply:
:facepalm: iirc I was looking at that as well; not sure how I missed it
I've updated the pr